data/gui/window/logging.cfg

```diff
@@ -0,0 +1,192 @@
+#textdomain wesnoth-lib
+###
+### Definition of the window to select logging options.
+### These first five things match the names from logging.cpp
+###
+#define _NAME
+"title_name"
+#enddef
+#define _DEBUG
+"debug"
+#enddef
+#define _INFO
+"info"
+#enddef
+#define _WARN
+"warn"
+#enddef
+#define _ERR
+"err"
+#enddef
```

Comment:
I'm not sure what the point of these macros are. Just why do you need them? Anyway, if they really are necessary, the `#enddef` should not start on a new line.